docs/guide/apm-release-notes.asciidoc

```diff
@@ -23,6 +24,67 @@ For a full list of changes, see the
 
 // end::notable-v8-highlights[]
 
+[[release-highlights-7.3.0]]
+=== APM version 7.3.0
+
+// tag::notable-v73-highlights[]
+
+[discrete]
+==== Elastic APM .NET Agent GA
+
+https://github.com/elastic/apm-agent-dotnet/[Elastic APM agent for .NET] is now
+generally available! The .NET Agent adds automatic instrumentation for ASP.NET
+Core 2.x+ and Entity Framework Core 2.x+, while also providing a
+{apm-dotnet-ref}/public-api.html[Public API] for the .NET agent that will allow
+you to instrument any .NET custom application code.
+
+[discrete]
+==== Aggregate service breakdown charts
+
+beta[] In addition to the transaction duration and throughput charts, the 7.3
+release adds aggregated service breakdown charts for services. These charts help
+you visualize where your application and services are spending most of their
+time, allowing you to get to the root cause of a performance problem quickly.
+These charts are available in Beta with support for certain APM agents(Java, Go,
+Node.js, Python).
+
+[role="screenshot"]
+image::images/apm-highlight-breakdown-charts.png[Aggregate service breakdown charts]
+
+[discrete]
+==== Agent sample rate configuration from APM UI
+
+beta[] Configuring sampling rate for your services is a whole lot easier with this
+release. The new settings page now lets you view and configure the sampling rate
+for your services from within the APM UI without changing the agent yaml files
```

Comment:
The agents don't have yaml config files. I suggest replace:

`for your services from within the APM UI without changing the agent yaml files
for each individual reporting instance of the service` ->
`for your services from within the APM UI without restarting them.` 
or something like that

Reply:
Done, thanks!